### ISSUE TITLE
Improve usability and accessibility of Unique Names

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/MyopicVicar#2358]
The page heading, "Unique names for people in …" is marked up as an `h3`: `h1` would seem appropriate (with text size adjusted if wished).

The headings "Surnames" and "Forenames" are also marked up as `h3`: headings should be hierarchical, so they should be marked up as `h2`.

The letter "headings" are marked up as paragraphs: they should be marked up as (actual) headings — `h3` would be appropriate.

There is no in-page navigation: suggest links to "Surnames" and "Forenames" as a minimum. Consider adding links to each letter and to "Additional Names".

A screen-reader user is likely to be unaware of the asterisks.

E0 U3 S6 C2: P11

Consider explaining the significance of asterisks and question marks on the page itself, or adding a link to the Help information.

**Comment by stsccfr** [comment]
Closing as not needed
